maskformatter/src/main/java/com/azimolabs/maskformatter/MaskFormatter.java

```diff
@@ -28,6 +31,21 @@ public MaskFormatter(String mask, EditText maskedField, char maskCharacter) {
         this.mask = mask;
         this.maskedField = maskedField;
         this.maskCharacter = maskCharacter;
+        this.passwordMask = getPasswordMask(maskedField);
+        setInputTypeBasedOnMask();
```

Comment:
Oops, this is from: https://github.com/AzimoLabs/MaskFormatter/pull/8.

Once that PR is merged this one will be in merge conflict and I'll fix this then :)